cmd/rookflex/cmd/expander.go

```diff
@@ -62,9 +63,9 @@ func handleExpandFs(cmd *cobra.Command, args []string) error {
 	var command *exec.Cmd
 	switch opts.FsType {
 	case "ext3", "ext4":
-		command = exec.Command("resize2fs", args[2])
+		command = exec.Command("resize2fs", args[2]) // #nosec G204
```

Comment:
Whenever we suppress the gosec error, let's also add a description, and also put it on the line before. For example:
```
// #nosec G204 Rook controls the input to the exec arguments
command = exec.Command("xfs_growfs", "-d", args[2])
```

Reply:
Done.